cmd/api/main.go

```diff
@@ -24,6 +24,8 @@ import (
 	"net/http"
 	"os"
 
+	_ "go.uber.org/automaxprocs"
```

Comment:
Could you also add the PR description as a comment here?

Reply:
Done